src/explorer/ISiteTreeItem.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export interface ISiteTreeItem {
+
+    defaultHostUrl: string;
```

Comment:
nit: Unless you're gonna add descriptions, I'd prefer there's no empty space between properties.  Just takes up a lot of space and makes the file look cluttered.